src/Spec2-Code/SpCodePresenter.class.st

```diff
@@ -197,7 +197,8 @@ SpCodePresenter >> browseSelectedSelectorIfGlobal: globalBlock ifNotGlobal: nonG
 	| variableOrClassName env |
 
 	variableOrClassName := self selectedSelector.
-
+	variableOrClassName ifNil: [ ^ self inform: 'No string could be parsed to answer your query.' ].
```

Comment:
in a second thought I do not like this. 
what to show if there are no results (or showing it at all) is a responsibility of the users of this.

Reply:
I think that's a good point.
However, given how the messages are done here, I'm very unsure about putting an "extra case" in the sender.
doBrowseSenders/Implementors just sends a message with two blocks and does not expect a result.

That would look like:
doBrowseSenders

    res := self
        browseSelectedSelectorIfGlobal: [ :global | 
            self systemNavigation browseAllSendersOf: global ]
        ifNotGlobal:  [ :selector |
            self systemNavigation browseAllReferencesTo: selector ].
       res ifNotNil: [  self inform: blablabla ]

I feel that's not better.
Moreover it creates a bit of code duplication for the two senders of this message.
Also, this is kind of a UI method already, isn't it?